src/components/GameList/GameList.tsx

```diff
@@ -124,7 +124,12 @@ export class GameList extends React.PureComponent<GameListProps, any> {
                 case 'move-number' :
                     lst.sort((a, b) => {
                         try {
-                            return a.json.moves.length - b.json.moves.length || a.id - b.id;
+                            if(a.json && b.json){
```

Comment:
The Shippable test failures have to do with the linter complaining about the lack of whitespace, on this line and [line 130](https://github.com/online-go/online-go.com/pull/218/files#diff-4a58b42a1aaf5f8daaffc4de471dc12aR1300).